spec/Section 3 -- Type System.md

```diff
@@ -302,17 +302,18 @@ enumerable. GraphQL offers an `Enum` type in those cases, where the type
 specifies the space of valid responses.
 
 Scalars and Enums form the leaves in response trees; the intermediate levels are
-`Object` types, which define a set of fields, where each field is another type
-in the system, allowing the definition of arbitrary type hierarchies.
+`Object` types, which define an ordered set of fields, where each field is
```

Comment:
Shouldn’t objects (and interfaces, and input objects) have an ordered _map_ of fields? The field names are unique, not the field definitions as a whole. `foo: Int, foo: String` would be a set of two different fields that happen to share a name.

Reply:
This is something I raised in the WG actually; but that change is beyond the scope of this PR.